### PR TITLE
feat: Implement the eBPF profile widget on dashboard

### PR DESCRIPTION
**Pull request by Fine0830**
Implement the eBPF profile widget on dashboard. Also close https://github.com/apache/skywalking/issues/8914

Videos

https://user-images.githubusercontent.com/20871783/164964676-48cbd032-a023-4272-8ed6-7275470dc047.mov

https://user-images.githubusercontent.com/20871783/164958055-97d90552-6f84-4640-9dac-d30312f1e24a.mov

https://user-images.githubusercontent.com/20871783/164960074-75d79152-4d5d-45f8-82a1-e3963ef9ef32.mov

 Signed-off-by: Qiuxia Fan <qiuxiafan@apache.org>

### CODE REVIEW

**Comment by wu-sheng** [comment]
The tip of task duration should be `The time range should not be out of 10 minutes`

**Comment by wu-sheng** [comment]
About SDK-profiling, is the endpoint selector supporting search?

The task list of ebpf profiling should be on the timestamp inverse order.

**Comment by Fine0830** [comment]
> About SDK-profiling, is the endpoint selector supporting search?
> 

Yes

**Comment by Fine0830** [comment]
> The tip of task duration should be `The time range should not be out of 10 minutes`

It's a response content. FYI @mrproliu

File: package.json [hunk 1]
```diff
@@ -12,11 +12,13 @@
   "dependencies": {
     "axios": "^0.24.0",
     "d3": "^7.3.0",
+    "d3-flame-graph": "^4.1.3",
     "d3-tip": "^0.9.1",
     "echarts": "^5.2.2",
     "element-plus": "^2.0.2",
     "lodash": "^4.17.21",
     "pinia": "^2.0.5",
+    "vis-timeline": "^7.5.1",
```
Comment (wu-sheng) [inline package.json:21]: Please add these to license file when sync UI.

**Review by wu-sheng — APPROVED** [review]
LGTM, @mrproliu Please fix words on the backend.

**Pull request merged**